lib/store_web/templates/starbook/index.html.eex

```diff
@@ -0,0 +1,71 @@
+    </div>
+  </section>
+
+  <section id="trackpad">
+    <div>
+      <h3>Slip and swipe.</h3>
+      <p>Glide across the large glass trackpad for precise and smooth multi-touch gestures. Scroll through documents or web pages, fly through your workspaces, and swipe to go back throughout elementary OS—all with 1:1 gestures that perfectly track your fingers.</p>
+    </div>
+    <figure>
+      <span>5.5&Prime;</span>
+    </figure>
+  </section>
+
+  <section id="keyboard">
+  <img alt="StarBook Mk V top-down with keyboard" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/keyboard-us.png") %>" />
+  <div>
+    <h3>A keyboard that’s pure function.</h3>
+    <p>Choose a US or UK layout featuring comfortable backlit keys with snappy scissor mechanisms. Tap a single key to switch between F1 through F12 or function keys that control volume, media, brightness, and more. Plus, an extra column of handy keys including Home, End, and Page Up/Down.</p>
+  </div>
+  </section>
+
+  <section id="privacy">
+    <img alt="StarBook Mk V webcam with illuminated privacy light" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/privacy.png") %>" />
+    <hr />
+    <h3>Ready for remote, with privacy built-in.</h3>
+    <p>The HD webcam is flanked by stereo microphones for clear video and voice calls. The bright white indicator shines any time the camera is on, while elementary OS lets you know any time the mics are being accessed.</p>
+  </section>
+
+  <section id="connectivity">
+    <img alt="StarBook Mk V side with ports visible" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/side.png") %>" />
+  <h3>Versatile connectivity.</h3>
+  <ul>
+    <li><strong>Wi-Fi 6</strong> and <strong>Bluetooth 5.1</strong> for all things wireless</li>
+    <li><strong>USB-C</strong> with <strong>Thunderbolt 4</strong>, power delivery, and display support for charging, accessories, and output all in one
+    <li><strong>Full-size USB 3</strong> on each side, and an extra USB 2 on the right</li>
+    <li><strong>3.5 mm headphone/<wbr>microphone jack</strong> for all the audio devices you need</li>
+    <li><strong>HDMI</strong> for easy output virtually everywhere</li>
+    <li><strong>microSD</strong> slot for expansion, mobile device file transfers, or flashing maker projects—you name it</li>
+  </ul>
+  </section>
+
+  <section id="elementary">
+    <img alt="StarBook Mk V head-on running elementary OS" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/hero.png") %>" />
+    <h2>Designed for elementary OS</h2>
+    <p>This isn’t just any laptop with elementary OS slapped on; Star Labs and elementary have partnered to ensure the best possible unity of hardware and software.  And with our ongoing partnership, both StarBook Mk V and elementary OS continue to get better, together.</p>
```

Comment:
I'm not sure this first line sets a good tone. I think we should focus more on explaining what makes it not just a laptop with an OS slapped on if that's an important part of the messaging

```suggestion
    <p>Star Labs and elementary have partnered to ensure the best possible unity of hardware and software.  And with our ongoing partnership, both StarBook Mk V and elementary OS continue to get better, together.</p>
```